requirements.txt

```diff
@@ -1,10 +1,10 @@
-django>=1.11,<3.0
+django>=2.2,<3.1
 django-model-utils
 jsonfield
 six
-netjsonconfig>=0.6.2,<0.7.0
-django-sortedm2m==2.0.0
-django-reversion>=2.0.6,<3.0.5
-django-x509>=0.5.0,<0.6.0
-django-taggit>=0.23.0,<0.24.0
-openwisp-utils>=0.2,<0.3
+netjsonconfig>=0.6.2,<0.8.0
+django-sortedm2m>=2.0.0,<3.1.0
+django-reversion>=2.0.6,<3.1.5
+django-x509>=0.6.0,<0.7.0
+django-taggit>=0.24.0,<1.3.0	
+openwisp-utils>=0.3,<0.4
```

Comment:
Update to `>=0.4` and it should be fixed! :smile:

Reply:
`>=0.4.0,<0.5.0`